src/mobile/src/main/java/tupperdate/android/appbars/mainTopBar.kt

```diff
@@ -1,27 +1,55 @@
 package tupperdate.android.appbars
 
-/*
-Commented because it doesn't find Modifier.weight, which however works in Onboarding.kt...
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.Icon
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import tupperdate.android.R
+import tupperdate.android.home.BarButton
+import tupperdate.android.home.normalButtonSize
+import tupperdate.android.ui.TupperdateTypography
 
 @Composable
-fun mainTopBar() {
-    TopAppBar(
-        title = {
-            Column(modifier = Modifier.weight(1f), horizontalGravity = Alignment.CenterHorizontally) {
-                Text(text = "tupper.date", textAlign = TextAlign.Center)
+fun mainTopBar(
+    onChatClick: () -> Unit,
+    onProfileClick: () -> Unit
+) {
```

Comment:
Missing a default Modifier.

Reply:
We don't need it. G-L told me to remove them in this case